sarracen/interpolate/interpolate.py

```diff
@@ -322,13 +335,16 @@ def interpolate_2d(data: 'SarracenDataFrame', target: str, x: str = None, y: str
     x_pixels, y_pixels = _set_pixels(x_pixels, y_pixels, x_min, x_max, y_min, y_max)
     _check_boundaries(x_pixels, y_pixels, x_min, x_max, y_min, y_max)
 
+    mass_data = _get_mass(data)
+    rho_data = _get_density(data)
+    w_data = data[target].to_numpy() * mass_data / rho_data
+
```

Comment:
There needs to be a check for if 'target' is density, right? Similar to interpolate_3d(), lines 550-555